Arrowgene.Ddon.GameServer/Handler/CharacterDecideCharacterIdHandler.cs

```diff
@@ -32,6 +33,15 @@ public override void Handle(GameClient client, IPacket packet)
             S2CCharacterDecideCharacterIdRes res = new S2CCharacterDecideCharacterIdRes();
             res.CharacterId = client.Character.CharacterId;
             res.CharacterInfo = new CDataCharacterInfo(client.Character);
+
+            if (Server.Database.GetCurrentMsqId(client.Character.CommonId) == QuestId.None &&
+                Server.Database.GetCompletedQuestsByType(client.Character.CommonId, QuestType.Main).Count == 0)
+            {
+                // TODO: Remove this in the future, it is needed for servers which keep progress before quests are added
+                Server.Database.InsertIfNotExistCompletedQuest(client.Character.CommonId, QuestId.ResolutionsAndOmens, QuestType.Main);
+                Server.Database.InsertQuestProgress(client.Character.CommonId, QuestId.TheSlumberingGod, QuestType.Main, 0);
+            }
```

Comment:
It should not loop back, because there will be MSQ progress. That said, I am not really firmly committed to this. I did this more as a convivence for the live server.

Reply:
Untested SQL, worth a shot:
```sql
replace INTO ddon_completed_quests
select character_common_id, 1, 1
from ddon_character;

replace INTO ddon_quest_progress
select character_common_id, 1, 2, 0
from ddon_character;
```

Reply:
Ok, I removed and blindly trusted your SQL and placed it in the PR description :)

Reply:
Gave it a quick test and it worked fine